### ISSUE TITLE
Add SignIn Page and SignUp Page

### ISSUE THREAD

**Issue by MageshSundarG** [Namanl2001/MERN-Gurujii-dev#145]
### Describe the feature you'd like to add 
Hey, In Mern-guruji-dev SignIn and SignUp Page is not there 
1.  A Login and Register Page with username and Password

### Describe how you will do it
I'll be creating a minimal full-stack login/authorization using the MERN stack. I’ll also integrate Redux for state management for our React components.

### Our app will allow users to

- [x] Register
- [x] Log in
- [x] Access protected pages only accessible to logged-in users
- [x] Stay logged in when they close the app or refresh the page
- [x] Log out

## Checklist

- [x] I've read the contribution guidelines. 

I would love to work on the above-mentioned issue. Assign Me!

**Comment by guptaji6489** [comment]
Any suggestion in this @Namanl2001  @sagar-barapatre

**Comment by Namanl2001** [comment]
IMO we don't need a separate sign in or signup page instead we can create some kind of dialogue box that would appear on clicking the sign in button. WDYT?

**Comment by MageshSundarG** [comment]
@Namanl2001 Yeah, It would be great to have a dialogue box rather than creating an entire page for sign-in and sign-up. 
Ok, Then
1. Let's create two buttons for sign-in and sign-up in the navbar in addition to GoogleSignin or
2. Let's merge those Oauth in the Sign-in and Sign-up ( Which is Recommended And Best practice for Websites )

Any Suggestions and I would like to work on this issue

**Comment by Namanl2001** [comment]
You can continue with that. thanks

**Comment by guptaji6489** [comment]
> @Namanl2001 Yeah, It would be great to have a dialogue box rather than creating an entire page for sign-in and sign-up.
> Ok, Then
> 
> 1. Let's create two buttons for sign-in and sign-up in the navbar in addition to GoogleSignin or
> 2. Let's merge those Oauth in the Sign-in dialogue box and Sign-up dialogue box ( Which is Recommended And Best practice for Websites )
> 
> Any Suggestions and I would like to work on this issue

Yeh, That's Great

**Comment by sagar-barapatre** [comment]
@MageshSundarG ??? It's been a long time.

**Comment by MageshSundarG** [comment]
I need Two Days Approx to finish, Stuck with other works. Finished the frontend Part. Working on the backend. Sry for the delay

**Comment by GuptaTanisha** [comment]
Can I work on this issue?

**Comment by ChitvanRamani22** [comment]
@GuptaTanisha are you woking on this.
